frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -18,7 +18,15 @@ describe('Localization', () => {
     cy.log('test masthead');
     cy.visit('/dashboards?pseudolocalization=true&lng=en');
     masthead.clickMastheadLink('help-dropdown-toggle');
-    cy.byTestID('application-launcher-item').isPseudoLocalized();
+    // wait for both console help menu items and additionalHelpActions items to load
+    // additionalHelpActions come from ConsoleLinks 'HelpMenu' yaml and are not translated
+    cy.get('.pf-c-app-launcher__group').should('have.length', 2);
```

Comment:
We should open a Bugzilla for the menu items not translated. I think it would be better to move this into console so they can be translated instead of leaving them as ConsoleLink resources, although it will require some operator changes.

Reply:
Created [Bug 1980531 - additionalHelpActions 'HelpMenu' ConsoleLinks not translated](https://bugzilla.redhat.com/show_bug.cgi?id=1980531)